src/shared/__tests__/routeUtils.test.ts

```diff
@@ -15,8 +15,8 @@ describe('routeUtils', () => {
 			expect(output).toEqual('/reset-password/tokenValue');
 		});
 		test('a non-parameterised path requires no object', () => {
-			const output = buildUrl('/consents/communication');
-			expect(output).toEqual('/consents/communication');
```

Comment:
Why was this changed and not appended? Just checkign we aren't losing any checks.